receivequeue.go

```diff
@@ -24,6 +24,7 @@ type receiveQueue struct {
 	readDeadline          time.Time
 	deadlineLock          sync.Mutex
 	closed                uint32 // 1 == true, 0 == false
+	unusedCPUAlignment    uint32 // Otherwise atomic alignments do not work on 32 bit arches
```

Comment:
I think you can also accomplish this by moving readFrameTip to the beginning of the struct which will be aligned in a predictable way.